mxcubecore/queue_entry/xrf_spectrum.py

```diff
@@ -111,7 +111,7 @@ def post_execute(self):
     def xrf_spectrum_status_changed(self, msg):
         """xrfSpectrumStatusChanged handler.
         Args:
-            msg (str): Message when xrfSpectrumStatusChanged emitted.
+            msg (str): Message when xrfSpectrumStatusChanged emited.
```

Comment:
```suggestion
            msg: Message when ``xrfSpectrumStatusChanged`` emitted.
```

Add the type hint annotation instead.